log/src/main/java/kcl/teamIndexZero/traffic/log/Outputs/Formatter_CSV.java

```diff
@@ -47,11 +44,13 @@ public String format(String origin_name, int log_level, Long log_number, Log_Tim
      */
     @Override
     public String format(String origin_name, Log_TimeStamp time_stamp, Long log_number, Exception e) {
-        StringWriter sw = new StringWriter();
-        PrintWriter pw = new PrintWriter(sw);
-        e.printStackTrace(pw);
-        String s = log_number.toString() + ";" + time_stamp.getDate() + ";" + time_stamp.getTime() + ";EXCEPTION;" + origin_name + "; " + sw.toString();
-        //TODO check exception details format
+        String s = String.format( "%d;%s;%s;EXCEPTION;%s;%s",
+                log_number,
+                time_stamp.getDate(),
+                time_stamp.getTime(),
+                origin_name,
+                e.toString()
```

Comment:
Are we escaping commas here? If I got the current code right, an excaption with message having a comma would break the CSV formatting (rows need to have same columns)